services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -76,6 +77,9 @@ public MigrationProject getMigrationProject(Long id)
     @Override
     public MigrationProject createMigrationProject(MigrationProject migrationProject)
     {
+        if (null != getProjectIdByName(migrationProject.getTitle()))
```

Comment:
yoda style we use now? :) Against it I'm not, but be consistent to make it easier to read the code, IMHO we should.